AdAway/src/main/res/values/strings.xml

```diff
@@ -105,6 +105,8 @@
     <string name="download_fail_title">Download failed</string>
     <string name="no_private_file">Private file can not be created.</string>
     <string name="no_private_file_title">Private file creation failed</string>
+    <string name="revert_systemless_successful">You disable the systemless mode.\nIt may be necessary to reboot Android for the changes to take effect.\n\nDo you want to reboot?\n(To prevent data loss make sure no app uses the sdcard at the moment!)</string>
```

Comment:
Thanks! I fixed it (and also change string key to be more relevant to the related action)